app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,10 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @new_comment = Comment.new(text: params[:comment][:text], author: current_user, post_id: params[:post_id].to_i)
+    redirect_to users_path if @new_comment.save
+  end
```

Comment:
- [Optional] It would be wise to add a private method for filtering your params to make your app even more secure. Please refer [here for more ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides) about param methods.